betterproto/__init__.py

```diff
@@ -1154,3 +1091,16 @@ async def _unary_stream(
             await stream.send_message(request, end=True)
             async for message in stream:
                 yield message
+
+
+from .lib.google.protobuf import (
```

Comment:
yes, i know :-/ it can't be at the top due to circular imports. the compiled google classes depend on the base classes in `__init__.py`, while init also needs the Google classes. Once we split up Init hopefully this can be avoided

Reply:
Oh dang, could you put a comment noting that it's to avoid circular imports?

Reply:
Sadly this was one of the reasons for the giant single file. Breaking everything up and resolving circular imports looked like significant work at the time so I put it off. Sorry!